client/my-sites/email/inbox-management/mailbox-selection-list/style.scss

```diff
@@ -0,0 +1,126 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
+	background-color: #fff;
+}
+
+.mailbox-selection-list {
+
+	.mailbox-selection-list__header {
+		.formatted-header__title {
+			font-size: $font-title-large;
+			line-height: $font-headline-medium;
+
+			@include break-large {
+				font-size: $font-headline-medium;
+			}
+		}
+
+		.formatted-header__subtitle {
+			font-size: $font-body;
+			line-height: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__status {
+		line-height: 24px;
+		font-size: $font-title-small;
+		position: fixed;
+		top: 50%;
+		left: 50%;
+		transform: translate( -50%, -50% );
```

Comment:
Candid question: can't we use flexbox instead of this transform to center an element both horizontally and vertically?

Reply:
I think we'd need fixed height or similar on the container for that to work? No?